python/alibiexplainer/setup.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import setup, find_packages
+
+tests_require = [
+    'pytest',
+    'pytest-tornasync',
+    'mypy'
+]
+
+setup(
```

Comment:
Should we be exposing this as a pip package? My thought is that this implementation can be dockerized and made available on our GCR as a standard explainer implementation, but perhaps shouldn't be something people are installing themselves. What do you think?

Reply:
I agree it will be mainly used as a provided image. I suppose that is true for the xgbserver and skserver as well which also have `setup.py`. For ease of development its use but I agree they probably won't be published to PyPi.